docs/src/man/basics.md

```diff
@@ -0,0 +1,2170 @@
+     │ Int64  Int64  String  Int64  String   Int64
+─────┼───────────────────────────────────────────────────
+   1 │     0     67  male        2  own               67
+   2 │     1     22  female      2  own               67
+   3 │     2     49  male        1  own               67
+   4 │     3     45  male        2  free              67
+   5 │     4     53  male        2  free              67
+   6 │     5     35  male        1  free              67
+   7 │     6     53  male        2  own               67
+   8 │     7     35  male        3  rent              67
+```
+
+We are swapping values of one column with other column:
+
+```jldoctest dataframe
+julia> transform(german, :Age => :Sex, :Sex => :Age) # swapping the value of `:Age` column with `:Sex` column
+1000×10 DataFrame
+  Row │ id     Age     Sex    Job    Housing  Saving accounts  Checking accoun ⋯
+      │ Int64  String  Int64  Int64  String   String           String          ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     0  male       67      2  own      NA               little          ⋯
+    2 │     1  female     22      2  own      little           moderate
+    3 │     2  male       49      1  own      little           NA
+    4 │     3  male       45      2  free     little           little
+    5 │     4  male       53      2  free     little           little          ⋯
+    6 │     5  male       35      1  free     NA               NA
+    7 │     6  male       53      2  own      quite rich       NA
+    8 │     7  male       35      3  rent     little           moderate
+  ⋮   │   ⋮      ⋮       ⋮      ⋮       ⋮            ⋮                ⋮        ⋱
+  994 │   993  male       30      3  own      little           little          ⋯
+  995 │   994  male       50      2  own      NA               NA
+  996 │   995  female     31      1  own      little           NA
+  997 │   996  male       40      3  own      little           little
+  998 │   997  male       38      2  own      little           NA              ⋯
+  999 │   998  male       23      2  free     little           little
+ 1000 │   999  male       27      2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+
+julia> df1 = german[:, [:Age, :Job]] # getting two columns
+1000×2 DataFrame
+  Row │ Age    Job
+      │ Int64  Int64
+──────┼──────────────
+    1 │    67      2
+    2 │    22      2
+    3 │    49      1
+    4 │    45      2
+    5 │    53      2
+    6 │    35      1
+    7 │    53      2
+    8 │    35      3
+  ⋮   │   ⋮      ⋮
+  994 │    30      3
+  995 │    50      2
+  996 │    31      1
+  997 │    40      3
+  998 │    38      2
+  999 │    23      2
+ 1000 │    27      2
+     985 rows omitted
+```
+
+We are applying transformation function `+` to our source columns `:Age` and `:Job` are passed as an arguments to transformation
```

Comment:
please either explain this example in more detail or remove it.

I would also add a reference to later sections of the manual giving more examples.

Reply:
> please either explain this example in more detail or remove it.
Can you suggest to me the right explanation for it? 

> I would also add a reference to later sections of the manual giving more examples.
Last time I had done but I had got a suggestion to remove and add some examples of `transform` 

but in another section, I have discussed the same functions for `GroupedDataFrame`.

Reply:
> but in another section, I have discussed the same functions for `GroupedDataFrame`.

I do not understand this sentence + where are these examples of `GroupedDataFrame` - I do not see them.

> Can you suggest to me the right explanation for it?

If you give more than one column to a transformation they are passed as positional arguments. So `[:Age, :Job] => (+) => :res` evaluates `+(df1.Age, df1.Job)` (which adds two columns) and stores the result in the `:res` column.

Reply:
> I do not understand this sentence + where are these examples of GroupedDataFrame - I do not see them.
I was saying in this PR i have not discussed transformation for `GroupedDataFrame`. When another PR will also be merged then I will add cross-reference here if required.